rofi-pass

```diff
@@ -19,6 +19,10 @@ USERNAME_field='user'
 AUTOTYPE_field='autotype'
 OTPmethod_field='otp_method'
 
+# Display these fields with entries.
+# NOTE: no default here, handled in code.
+SHOWFIELDS_field='showfields'
```

Comment:
Typo? Shouldn't that be `show_fields`?

Reply:
No, it's the name in the entry, i.e. you can have `showfields: autotype` to only get "autotype" in the dialog.
`show_fields` is the var that rofi-pass uses to display them.

Reply:
Please update the documentation accordingly, as it makes it easier to grasp the concept :wink:

Reply:
Will do - waiting for some approval that this is good in general first.

It would go to README.md then, right?

Reply:
Yep, README.md is the documentation.